### PR TITLE
NEXUS-8363 - Add repository URL column to the repository list

### PR DESCRIPTION
**Pull request by kellyrob99**
- include url to the (not yet implemented) browse feature in the tabular view
- set default target for links to _blank, which is reasonable given that we no longer open tabs in the NX window

screen shot 2015-04-06 at 10 52 51 am

### CODE REVIEW

File: plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/repository/RepositoryList.js [hunk 1]
```diff
@@ -38,8 +38,16 @@ Ext.define('NX.coreui.view.repository.RepositoryList', {
     { header: NX.I18n.get('ADMIN_REPOSITORIES_LIST_NAME_COLUMN'), dataIndex: 'name', flex: 1 },
     { header: NX.I18n.get('ADMIN_REPOSITORIES_LIST_TYPE_COLUMN'), dataIndex: 'type' },
     { header: NX.I18n.get('ADMIN_REPOSITORIES_LIST_FORMAT_COLUMN'), dataIndex: 'format' },
-    { header: NX.I18n.get('ADMIN_REPOSITORIES_LIST_STATUS_COLUMN'), dataIndex: 'status', flex: 1 }
+    { header: NX.I18n.get('ADMIN_REPOSITORIES_LIST_STATUS_COLUMN'), dataIndex: 'status', flex: 1 },
+    { header: NX.I18n.get('ADMIN_REPOSITORIES_LIST_URL_COLUMN'), dataIndex: 'url', xtype: 'nx-linkcolumn', flex: 1 }
   ],
+  listeners: {
+    cellclick: function(grid, td, cellIndex, record, tr, rowIndex, e) {
+      if(e.getTarget('a')) {
```
Comment (adreghiciu) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/repository/RepositoryList.js]: Don't we want this in all cases for a link in a drill down grid?
So maybe we want this functionality moved there

**Comment by adreghiciu** [comment]
+1

File: plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/repository/RepositoryList.js (hunk 1 above)
Comment (kellyrob99) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/repository/RepositoryList.js]: Good point, will look into it.

**Comment by djsauble** [comment]
“Url” should be all uppercase since it’s an acronym.

+1 otherwise.

**Comment by kellyrob99** [comment]
@djsauble  08e630a15102bfbee1aefc3d084d05f8c6b5bb91

**Pull request merged**